client/packages/programs/src/JsonForms/components/DateRange.tsx

```diff
@@ -78,7 +78,9 @@ const UIComponent = (props: ControlProps) => {
             onChange={date => updateDate('afterOrEqualTo', date)}
             disabled={!props.enabled}
             actions={actions}
-            maxDate={DateUtils.getDateOrNull(data?.to) ?? undefined}
+            maxDate={
+              DateUtils.getDateOrNull(data?.beforeOrEqualTo) ?? undefined
```

Comment:
Fix because this actually isn't working... Also makes me question if the encounters report is actually filtering out correctly? Doesn't appear to be.

Reply:
Oh rip, how was that working 👀 

I think DateRange has value - we pretty often need a date range filter on a report. The component handles the max/min logic in one place, so we don't need to keep implementing on the JSON schema

Reply:
Hmm, it's only used in the Encounters though... I've used the DateTime one for this report, works just the same~